### PR TITLE
Column number test DO NOT MERGE

### PR DESCRIPTION
**Pull request by njacazio**
@ktf this PR is to test the compilation.
With this code I am unable to compile on Ubuntu 18.04 as I get:
```
DEBUG:O2:O2:0: /home/njacazio/alice/sw/SOURCES/O2/v1.2.0/0/Analysis/Tasks/trackselection.cxx:316:76:   required from here
DEBUG:O2:O2:0: /usr/include/c++/7/type_traits:1070:12: fatal error: template instantiation depth exceeds maximum of 900 (use -ftemplate-depth= to increase the maximum)
DEBUG:O2:O2:0:      struct __is_direct_constructible_new
DEBUG:O2:O2:0:             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
DEBUG:O2:O2:0: compilation terminated.
```
As discussed this morning I added two rows to the collision table (1 collision time) and 7 rows (not all the 9 expected times) to the tracksextra table.
This is enough to encounter the issue.
Let's see what the tests will tell.

### CODE REVIEW

File: Framework/Core/include/Framework/AnalysisDataModel.h [hunk 1]
```diff
@@ -52,9 +52,40 @@ DECLARE_SOA_COLUMN(NumContrib, numContrib, uint32_t);
...
+DECLARE_SOA_COLUMN(CollisionTime5, collisionTime5, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes5, collisionTimeRes5, float);
+DECLARE_SOA_COLUMN(CollisionTime6, collisionTime6, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes6, collisionTimeRes6, float);
+DECLARE_SOA_COLUMN(CollisionTime7, collisionTime7, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes7, collisionTimeRes7, float);
+DECLARE_SOA_COLUMN(CollisionTime8, collisionTime8, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes8, collisionTimeRes8, float);
+DECLARE_SOA_COLUMN(CollisionTime9, collisionTime9, float);
+DECLARE_SOA_COLUMN(CollisionTimeRes9, collisionTimeRes9, float);
```
Comment (ktf) [inline Framework/Core/include/Framework/AnalysisDataModel.h:74]: This you can of course change to be an array column. Any reason you did not do so?

**Comment by ktf** [comment]
So it seems to be some GCC quirk, given on macOS it compiles. Investigating.

File: Framework/Core/include/Framework/AnalysisDataModel.h (hunk 1 above)
Comment (njacazio) [reply Framework/Core/include/Framework/AnalysisDataModel.h:74]: Yes, you are right, I wanted to have the PR open with the code that I had locally.
I do not intend to have those added to the AF, but I agree that if I wanted to add them I would use an array, as you suggested yesterday.

Comment (ktf) [reply Framework/Core/include/Framework/AnalysisDataModel.h:74]: Ok. Shall we do that to unblock you at least on that part? We can keep the other as is to track the issue. I did some investigations yesterday and indeed it seems to be some issue in the implementation of `std::tuple<>` in GCC. If you accept the PR I did to your branch (or you allow me to edit this) we might be able to get more insights.

Comment (njacazio) [reply Framework/Core/include/Framework/AnalysisDataModel.h:74]: Hi @ktf, I should have merged the PR you opened on my branch, however you started from the master branch while I included your change into the dev version.
In principle the PR is already open to edits by reviewers.

**Comment by ktf** [comment]
Seems to have fixed it.

**Comment by njacazio** [comment]
@ktf , I agree, I'm also able to compile locally. Can I close this PR?

**Comment by ktf** [comment]
Great! Thanks for the confirmation!

**Pull request closed without merging**